pkgs/by-name/ph/phase-cli/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "phase-cli";
+  version = "1.18.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "phasehq";
+    repo = "cli";
+    tag = "v${version}";
+    hash = "sha256-4DvvTIXTV10euFSFpULcomrAhfZbOBZ0MKb5oR9GX64=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    keyring
+    questionary
+    cffi
+    requests
+    pynacl
+    rich
+    pyyaml
+    toml
+    python-hcl2
+  ];
+
+  nativeCheckInputs = [
+    versionCheckHook
+    python3Packages.pytestCheckHook
+  ];
+
+  pytestFlagsArray = [
+    "tests/crypto.py"
+    "tests/phase_run.py"
+    "tests/secret_referencing.py"
+    "tests/secrets_export.py"
+    "tests/tags.py"
```

```suggestion
    "tests/*.py"
```
is what upstream does.